From: kjlau0112 <lkarnjye@gmail.com>
Date: Sun, 20 Mar 2022 16:53:23 +0800
Subject: [PATCH] [Stargazer OS] U-boot development enablement

Again i'm missing C programming and want something fancy
in resume.Can't believe i need git reset because i missed
a space.

Signed-off-by: kjlau0112 <lkarnjye@gmail.com>
---
 board/st/stm32mp1/stm32mp1.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/board/st/stm32mp1/stm32mp1.c b/board/st/stm32mp1/stm32mp1.c
index 42979d80f5..6844a23feb 100644
--- a/board/st/stm32mp1/stm32mp1.c
+++ b/board/st/stm32mp1/stm32mp1.c
@@ -115,6 +115,7 @@ int checkboard(void)
 	if (fdt_compat && fdt_compat_len)
 		printf(" (%s)", fdt_compat);
 	puts("\n");
+	printf("Stargazer OS U-boot Development Enablement\n");
 
 	/* display the STMicroelectronics board identification */
 	if (CONFIG_IS_ENABLED(CMD_STBOARD)) {
-- 
2.17.1

